### ISSUE TITLE
Help with building process

### ISSUE THREAD

**Issue by americoperez49** [maximegris/angular-electron#616]
Hi guys,

I'm new to all of this so my apologies if i'm missing something obvious.

I am trying to use the angular-electron repo in combination with the [Devextreme Layout](https://js.devexpress.com/Documentation/Guide/Common/DevExtreme_CLI/#DevExtreme_Application/Add_the_DevExtreme_Layout_to_an_Existing_Application) but I cant seem to get it to work.

I tried running the command `devextreme add devextreme-angular` after cloning the angular-electron repo but it errors out towards the end and never completes. I get the following error
 `Cannot read property 'toString' of null 1`

I also tried to build the Devextreme layout app in a different repo and then tried to copy files/folders over to the angular-electron repo but I also get errors when building. The errors are webpack related. I can post if needed.

Steps to reproduce the error:
1. Clone angular-electron repo
2. npm install
3. npm i -g devextreme-cli
4. devextreme add devextreme-angular

Other Info that might be important:
[Install DevExtreme](https://js.devexpress.com/Documentation/Guide/Angular_Components/Getting_Started/Add_DevExtreme_to_an_Angular_CLI_Application/)

Thanks for anyone's help in advance!

**Comment by americoperez49** [comment]
I managed to get the command to work by moving the location of the tsconfig file, but now i just get the same webpack build errors as before.

```
｢wdm｣: HookWebpackError: ENOENT: no such file or directory, scandir
    at makeWebpackError (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\webpack\lib\HookWebpackError.js:49:9)
    at C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\webpack\lib\Compilation.js:2496:12
    at eval (eval at create (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:31:1) 
    at fn (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\webpack\lib\Compilation.js:427:17)
    at _next3 (eval at create (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:28:1)
    at eval (eval at create (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:46:1) 
    at fn (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\webpack\lib\Compilation.js:439:9)
    at _next2 (eval at create (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:41:1)
    at eval (eval at create (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:59:1) 
    at eval (eval at create (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:11:1) 
    at C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\webpack\lib\SourceMapDevToolPlugin.js:546:10
    at C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\neo-async\async.js:2830:7
    at Object.each (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\neo-async\async.js:2857:9)
    at C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\webpack\lib\SourceMapDevToolPlugin.js:382:17
    at C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\neo-async\async.js:2830:7
    at Object.each (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\neo-async\async.js:2857:9)
-- inner error --
Error: ENOENT: no such file or directory, scandir
    at Object.readdirSync (fs.js:1043:3)
    at CacheBackend.provideSync (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\webpack\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:233:32)
    at WebpackFileSystem.listPaths (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\WebpackFileSystem.js:78:24)
    at LicenseTextReader.readLicense (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\LicenseTextReader.js:37:54)
    at PluginChunkReadHandler.processModule (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\PluginChunkReadHandler.js:46:62)
    at C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\PluginChunkReadHandler.js:22:23
    at WebpackModuleFileIterator.internalCallback (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\WebpackModuleFileIterator.js:25:13)
    at WebpackModuleFileIterator.iterateFiles (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\WebpackModuleFileIterator.js:10:9)
    at C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\PluginChunkReadHandler.js:20:32
    at WebpackChunkModuleIterator.iterateModules (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\WebpackChunkModuleIterator.js:42:21)
    at PluginChunkReadHandler.processChunk (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\PluginChunkReadHandler.js:19:29)
    at _loop_1 (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\WebpackCompilerHandler.js:92:37)
    at WebpackCompilerHandler.iterateChunks (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\WebpackCompilerHandler.js:127:17)
    at C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\WebpackCompilerHandler.js:43:31
    at fn (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\webpack\lib\Compilation.js:425:10)
    at _next3 (eval at create (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:28:1)
× ｢wdm｣: caused by plugins in Compilation.hooks.processAssets
Error: ENOENT: no such file or directory, scandir
    at Object.readdirSync (fs.js:1043:3)
    at CacheBackend.provideSync (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\webpack\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:233:32)
    at WebpackFileSystem.listPaths (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\WebpackFileSystem.js:78:24)
    at LicenseTextReader.readLicense (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\LicenseTextReader.js:37:54)
    at PluginChunkReadHandler.processModule (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\PluginChunkReadHandler.js:46:62)
    at C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\PluginChunkReadHandler.js:22:23
    at WebpackModuleFileIterator.internalCallback (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\WebpackModuleFileIterator.js:25:13)
    at WebpackModuleFileIterator.iterateFiles (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\WebpackModuleFileIterator.js:10:9)
    at C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\PluginChunkReadHandler.js:20:32
    at WebpackChunkModuleIterator.iterateModules (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\WebpackChunkModuleIterator.js:42:21)
    at PluginChunkReadHandler.processChunk (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\PluginChunkReadHandler.js:19:29)
    at _loop_1 (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\WebpackCompilerHandler.js:92:37)
    at WebpackCompilerHandler.iterateChunks (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\WebpackCompilerHandler.js:127:17)
    at C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\license-webpack-plugin\dist\WebpackCompilerHandler.js:43:31
    at fn (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\webpack\lib\Compilation.js:425:10)
    at _next3 (eval at create (C:\Users\americo.perez\Downloads\Repos\angular-electron\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:28:1)
```

**Comment by americoperez49** [comment]
I managed to figure it out.

Basically, I created a devextreme application from scratch without electron.
Then I added electron to it after the fact and installed all of the dependencies that were required to run the command 
`npm run start` accoding to what was in the package.json of angular-electron project.

I then used the package.json commands that were in the angular-electron repo in combination with the debugging files in the .vscode folder as a guide to get Debugging working.
